index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta charset="utf-8">
-    <title>"Hello Microverse"</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>"Tasnimul Hasan Portfolio"</title>
```

Comment:
- [ ] Kindly name the `title `according to the project. Here it should be `Hello Microverse` so that the content of the project is clear with relevant title.